distributed/worker.py

```diff
@@ -2646,8 +2598,73 @@ def transition_released_forgotten(
         self.tasks.pop(ts.key, None)
         return recommendations, []
 
+    # {
+    #     (start, finish):
+    #     transition_<start>_<finish>(
+    #         self, ts: TaskState, *args, stimulus_id: str
+    #     ) -> (recommendations, instructions)
+    # }
+    TRANSITIONS_TABLE: ClassVar[
```

Comment:
Meta question: Do we have any conventions on the ordering of class variable definitions, instance variable declarations, class/instance method definitions, etc?

Reply:
We don't. In this case the definition MUST be after the transitions though.

Reply:
Makes sense, I was just wondering how strict we'd be here, also when it comes to ordering of public/private, class/instance methods. It looks like the rule there is "whatever seems to be a reasonable order/grouping".

Reply:
We don't group public and private separately. e.g. typically we put private helper methods next to their public parent methods.